lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      #  Checks for empty before and after hooks.
+      #
+      # @example
+      #   # bad
+      #   before {}
+      #   after do; end
+      #   before(:all) do
+      #   end
+      #   all(:all) { nil }
+      #
+      #   # good
+      #   before { create_users }
+      #   after do; cleanup_users; end
+      #   before(:all) do
+      #     create_feed
+      #   end
+      #   all(:all) { cleanup_feed }
+      class EmptyHook < Cop
+        MSG = 'Empty hook detected.'
+
+        def_node_matcher :empty_hook?, <<~PATTERN
+          (block
+            $(send nil? {:before :after}
```

Comment:
There are some aliases and rarely used hooks, all defined [here](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb#L87). I understand that `after` is quite different, but wouldn't it be bad to detect an empty around (without `yield`) along the way?